lonestar/analytics/cpu/k-core/README.md

```diff
@@ -17,22 +17,21 @@ its neighbors as it is considered removed from the graph.
 INPUT
 --------------------------------------------------------------------------------
 
-Takes in **symmetric** Galois .gr graphs. The results obtained
```

Comment:
In the description above this: "_A k-core is defined as a subgraph of a 
graph in which all vertices of degree less than k have been removed from the 
graph. The remaining nodes must all have a degree of at least k._" 

Can be changed to: "_A k-core of a graph is defined as a subgraph in which all vertices have a degree of at least k._"

Reply:
Agreed. To be precise (from wiki), "A k-core of a graph G is a maximal connected subgraph of G in which all vertices have degree at least k".